### PR TITLE
Fix #429

### LINKED ISSUE #429: Add snapshot mode
**Is your feature request related to a problem? Please describe.**
Today, when the user needs to run suzieq poller just once to gather data, say for network validation, they have a more complex workflow to execute. They have to first run the poller with --run-once=gather option followed by another run of suzieq poller with -i option on the directory the gather was done.

It'd be useful to add a mode called --snapshot that takes the option of running once but writing to parquet directly thereby avoiding the two-step workflow described above. 

It'd be even more useful if the snapshot mode could take a timeout parameter to try executing the command exactly once for each host in the inventory, but add a timeout to give up trying.

**Describe the solution you'd like**
--run-once=gather was a option addded to Suzieq to make it easier for the dev team to support poller parsing issues in the field, and to develop reproducible data for tests. 

**Additional context**

### PR DESCRIPTION
**Pull request by ddutt**
Add a snapshot mode to poller. This has multiple scenarios in which it is useful.

### CODE REVIEW

File: suzieq/poller/worker/services/service.py [hunk 1]
```diff
@@ -813,6 +813,11 @@ async def run(self):
             # Post a cmd to fire up the next poll after the specified period
             self.logger.debug(
                 f"Rescheduling service for {self.name} service")
+            if self.run_once == "update":
```
Comment (claudiolor) [inline suzieq/poller/worker/services/service.py:816]: Just a detail: logging `Rescheduling service` when the mode is `update` is not correct

**Review by claudiolor — APPROVED** [review]

File: suzieq/poller/worker/services/service.py [hunk 2]
```diff
@@ -761,7 +761,7 @@ async def run(self):
                 await self.commit_data(result, output[0]["namespace"],
                                        hostname)
             else:
```
Comment (claudious96) [inline suzieq/poller/worker/services/service.py:763]: Just a small comment:
```suggestion
            elif  self.run_once in ["gather", "process", "update"]:
```

**Review by claudious96 — APPROVED** [review]

File: suzieq/poller/worker/services/service.py (hunk 1 above)
Comment (ddutt) [reply suzieq/poller/worker/services/service.py:816]: Ack, good catch. FIxing it.

File: suzieq/poller/worker/services/service.py (hunk 2 above)
Comment (ddutt) [reply suzieq/poller/worker/services/service.py]: Ack

**Pull request merged**